admin/app/components/campaigns/update.js

```diff
@@ -10,7 +10,7 @@ const Validations = buildValidations({
     validators: [
       validator('presence', {
         presence: true,
-        ignoreBlank: true,
+        ignoreBlank: false,
```

Comment:
**question** : cet attribut sert à quoi ?

Reply:
l'ignoreBlank attribute vérifie s'il y a des espace  ou pas puis les accepter ou pas. pour cet validateur qui a `presence: true,
        ignoreBlank: false,` on vérifie que le nom existe tout le temps et ne contient pas un espace.
        le message doit être  "'should not be empty or consist only of spaces'"